### DISCUSSION TITLE
MiXCR, iRepertoire and IMDA pipeline

### DISCUSSION THREAD

**Discussion by ubertramp** [milaboratory/mixcr#1717] [Q&A] [answered]
Hello.  I apologize if this is the wrong place to ask this. First timer.

I am working with the IMDA repertoire pipeline to analyze CDR3 data generated by iRepertoire.  I have both BCR and TCR data but I want to focus first on the BCR data.  I stumbled across the presets page and saw that there is an iRepertoire preset.  I'm hoping I can get it to work with IMDA, but that pipeline uses an old version of MiXCR.  Specifically MiXCR 3.0.13.  Right now, the way I'm calling MiXCR is through a command string that starts off with this:

`mixcr = "java -Xmx8G -jar " + My_Path + "/MiXCR/mixcr-3.0.13/mixcr.jar"

'mixcr_library = "--library imgt.201802-5.sv2.json" `

My question is, will the presets work with that version of MiXCR?  And if yes, how do I modify the code above to use it?

Thanks!

EDIT. I have both arm_Rx.fq.gz and dam_Rx.Fq.gz files from iRep.

**Comment by mizraelson** [comment, marked as answer]
Hi,
to use the presets you would need to install the latest version of mixcr available from [here](https://github.com/milaboratory/mixcr/releases/).
